airbyte-integrations/connectors/source-shopify/source_shopify/scopes.py

```diff
@@ -21,6 +21,7 @@
     "CustomerAddress": ("read_customers",),
     # SCOPE: read_orders
     "Orders": ("read_orders",),
+    "CustomerJourneySummary": ("read_orders",),
```

Comment:
[The documentation](https://shopify.dev/docs/api/admin-graphql/2024-04/objects/CustomerJourneySummary#top) mentions that `read_marketplace_orders` is also fine. Do we need to reflect that here?

Reply:
No, since this is the additional scope we need to have, but it's not mandatory to have.